.github/workflows/upload-annotations.yml

```diff
@@ -0,0 +1,35 @@
+name: Upload Annotations
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  upload-annotations:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node and pnpm
+        uses: logto-io/actions-node-pnpm-run-steps@v1.0.2
+
+      - name: Prepack
+        run: pnpm prepack
+
+      - name: Lint with Report
+        run: pnpm -- lerna run --parallel lint:report && node merge-eslint-reports.js
+
+      - name: Annotate Code Linting Results
+        uses: ataylorme/eslint-annotate-action@1.2.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          only-pr-files: false
```

Comment:
will it be more user friendly if set to only-pr-files true

Reply:
- one of the purposes of this PR is to catch legacy warnings
- actually this param doesn't work for now... will remove

Reply:
updated